.travis.yml

```diff
@@ -72,6 +72,5 @@ matrix:
       - echo "$DOCKER_PASS_QUAY" | docker login -u "$DOCKER_USER_QUAY" --password-stdin quay.io;
       - docker pull quay.io/codait/max-base:powerai-ppc64le-"$VERSION";
       - docker pull quay.io/codait/max-base:powerai-x86_64-"$VERSION";
-      - docker manifest create quay.io/codait/max-base:powerai-"$VERSION" quay.io/codait/max-base:powerai-ppc64le-"$VERSION" quay.io/codait/max-base:powerai-x86_64-"$VERSION";
-      - docker manifest push quay.io/codait/max-base:powerai-"$VERSION";
+      - docker manifest create quay.io/codait/max-base:powerai-ppc64le-"$VERSION" quay.io/codait/max-base:powerai-x86_64-"$VERSION";
```

Comment:
I'm wondering whether we should completely remove this "manifest" section. According to the [document](https://docs.docker.com/engine/reference/commandline/manifest/):

- Manifest is experimental, not ready for production (yet).
- Manifest is mostly for supporting multiple archs. But since we have removed the key manifest, I'm wondering whether the whole manifest section makes sense any more.

Reply:
Good point, we can look into this in the future once it matures